frontend/public/components/dashboards-page/overview-dashboard/details-card.tsx

```diff
@@ -89,6 +88,14 @@ export const DetailsCard_ = connect(mapStateToProps)(({
         <DetailsBody>
           {openshiftFlag ? (
             <>
+              <DetailItem
+                key="apiurl"
```

Comment:
can you remove `key` prop for all Items ? its actually not needed

Reply:
done